scripts/install-filecoin-parameters.sh

```diff
@@ -10,5 +10,6 @@ generate_params() {
   RUST_LOG=info ./proofs/bin/paramcache --test-only
 }
 
-fetch_params
+# Disabling paramfetch until optimizations can be completed
```

Comment:
@ognots - `paramcache --test-only` will generate Groth parameters and verifying keys for 1024-byte sectors only. This means that, if this PR is merged, developers (i.e. someone running `deps`) will not be able to verify proofs generated over 256MiB sectors.

I strongly suggest that if you disable `fetch_params` that you remove the `--test-only` flag.

Reply:
@laser I believe there was already a decision made to _omit_ the large proof parameters from the `deps` function, as they are not needed to run tests, which is the primary use case. @anorth @travisperson probably have more context on this.

For anyone wanting to prove large sector size, the Wiki's _Getting Started_ page will instruct users to invoke `paramcache` with no args, as you suggest.

Reply:
also see https://github.com/filecoin-project/go-filecoin/pull/3375

Reply:
@ognots I think the issue is that we want verifying keys for large sectors as well, and paramcache does not support a way to generate just the verifying keys.

Reply:
@travisperson 

> [..] we want verifying keys for large sectors as well, and paramcache does not support a way to generate just the verifying keys

Yep, that's correct.